var/spack/repos/builtin/packages/bml/package.py

```diff
@@ -38,12 +38,23 @@ class Bml(CMakePackage):
     version('develop', git='https://github.com/lanl/bml', branch='master')
 
     variant('shared', default=True, description='Build shared libs')
+    variant('mpi', default=False, description='Build with MPI Support')
```

Comment:
@adamjstewart: Should the default be `True`?

Reply:
Yeah, probably. It only becomes a problem when packages with an MPI variant depend on other packages with MPI variants. We "fixed" this by setting the default to always be True. Hopefully the new concretizer will be able to handle this.